sklearn/model_selection/_validation.py

```diff
@@ -24,13 +24,190 @@
 from ..utils.validation import _is_arraylike, _num_samples
 from ..utils.metaestimators import _safe_split
 from ..externals.joblib import Parallel, delayed, logger
-from ..metrics.scorer import check_scoring
+from ..externals.six.moves import zip
+from ..metrics.scorer import check_scoring, _check_multimetric_scoring
 from ..exceptions import FitFailedWarning
 from ._split import check_cv
 from ..preprocessing import LabelEncoder
 
-__all__ = ['cross_val_score', 'cross_val_predict', 'permutation_test_score',
-           'learning_curve', 'validation_curve']
+
+__all__ = ['cross_validate', 'cross_val_score', 'cross_val_predict',
+           'permutation_test_score', 'learning_curve', 'validation_curve']
+
+
+def cross_validate(estimator, X, y=None, groups=None, scoring=None, cv=None,
+                   n_jobs=1, verbose=0, fit_params=None,
+                   pre_dispatch='2*n_jobs', return_train_score=False):
```

Comment:
Is it confusing that `return_train_score=True` by default in `*SearchCV`? Consistency here would be one of the most important factors.